config/jjb-templates/project-mojo-matrix.yaml

```diff
@@ -917,6 +939,8 @@
           - "xenial-pike"
           - "artful-pike"
           - "bionic-queens"
```

Comment:
Duplicate entry.  Not blocking on that, though.  I will fix that as a trivial follow-up so we can move along.  Thank you.